packages/lib/src/components/paywall/Renderer.svelte

```diff
@@ -75,8 +75,6 @@
         items: [item],
         discountCode: product.discountCode,
         redirectUrl: userProps?.['redirect-url'] || window.location.href,
-        price: product.price,
-        currency,
```

Comment:
I think we dont need to send and price and currency when creating the checkout, since the checkout will pull that from the catalog anyway right?

Reply:
It's used to create a hash that is compared with the existing value in the checkout, so we know if there's been a change in the price. I'm not sure where this hash should be created.. In a way I think it might be better to post the price and currency all the way to the checkout so we don't have this logic in multiple places?

Reply:
@markusahlstrand do you mean that we want to use it to create this hash? 🙂 because i cant see this hash being generated in sesamy-js or the checkout-api now? Looks like sesamy-js is just forwarding it and checkout-api does not have price and currency in the type for the body..

@majkiee any thoughts on this?

Reply:
Humm let's check tomorrow but the checkout does not care about the incoming price but we do send a hash to the catalog which tells it if it needs to update the price or not. Can't remember how we generate it though

Reply:
I think we should pass the price and currency to the proxy at least. It feels dangerous to have the hashing-algorithm as part of the client. I really think that algorithm should be internal to the catalog and not exposed at all, but for now we can do the hashing in the proxy.

Reply:
OK but im merging this as is then, price and currency is being sent to the createCheckout method, and i let you guys figure out this hashing thing 🙃